components/chainhook-cli/src/cli/mod.rs

```diff
@@ -23,7 +23,7 @@ use chainhook_sdk::chainhooks::types::{
 };
 use chainhook_sdk::types::{BitcoinNetwork, BlockIdentifier, StacksNetwork};
 use chainhook_sdk::utils::{BlockHeights, Context};
-use clap::{Parser, Subcommand};
+use clap::{ErrorKind, Parser, Subcommand};
```

Comment:
Unused `ErrorKind` import. Can we remove it?

Reply:
Done, thanks!